src/js/view/controls/components/menu/menu-item.js

```diff
@@ -5,7 +5,7 @@ import { itemRadioButtonTemplate, itemTemplate } from 'view/controls/templates/m
 export class MenuItem {
     constructor(_content, _action, _template = itemTemplate) {
         this.el = createElement(_template(_content));
-        this.ui = new UI(this.el, { directSelect: true }).on('click tap enter', _action, this);
+        this.ui = new UI(this.el).on('click tap enter', _action, this);
```

Comment:
don't we need to revert this as well: https://github.com/jwplayer/jwplayer/pull/3552/files#diff-2e6b6101310fae49863b21879469bd66R36

Reply:
Nope, only here. If we ever add text to category buttons, or other children that we want the event to bubble to we will have to revert that line as well.